downstream/modules/platform/proc-controller-remediate-insights-inventory.adoc

```diff
@@ -2,31 +2,31 @@
 
 [id="controller-remediate-insights-inventory"]
 
-= Remediating a Red Hat Insights inventory
+= Remediating a Red Hat Lightspeed inventory
 
 [role="_abstract"]
-Remediation of a Red Hat Insights inventory enables {ControllerName} to run Red Hat Insights playbooks with a single click.
+Remediation of a {RHLightspeed} inventory enables {ControllerName} to run {RHLightspeed} playbooks with a single click.
 
-You can do this by creating a job template to run the Red Hat Insights remediation.
+You can do this by creating a job template to run the {RHLightspeed} remediation.
 
 .Procedure
 . From the navigation menu, select {MenuAETemplates}.
 . On the *Templates* list view, click btn:[Create template] and select from the list.
 . Enter the appropriate details in the following fields.
```

Comment:
The sentence that follows this one seems like an admonition (Note that the following fields...). Suggestion: 

```suggestion
. Enter the appropriate details in the following fields, all of which require {RHLightspeed} related entries.
```